sdk/ai/azopenai/example_client_audio_test.go

```diff
@@ -120,3 +120,49 @@ func ExampleClient_GenerateSpeechFromText() {
 
 	// Output:
 }
+
+func ExampleClient_AudioTranslation() {
+	openAIKey := os.Getenv("OPENAI_API_KEY")
```

Comment:
```suggestion
	azureOpenAIKey := os.Getenv("AZURE_OPENAI_API_KEY")
```

Previous mistake on my part there.

Reply:
done